app/models/decidim/time_tracker/assignee_questionnaire.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module Decidim
+  module TimeTracker
+    class AssigneeQuestionnaire < ApplicationRecord
```

Comment:
is this for creating questionnaires for gender/age for the assignee?
I'm not sure if this information should simply be 2 fields in the assignee table. Let's discuss this

Reply:
Yes, that's probably the most wise approach right now. We can always add something like this later.